spec/models/legislator_spec.rb

```diff
@@ -227,7 +227,11 @@
 
   describe "#serializable_hash" do
     let(:senator) { FactoryGirl.create(:senator) }
-    let(:keys) { ["id", "chamber", "party", "state_rank", "name", "title", "image_url", "display_district", "state_abbrev", "district_code"] }
+    let(:keys) { [
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.